regulations/views/preamble.py

```diff
@@ -47,14 +47,15 @@ def get(self, request, *args, **kwargs):
             raise Http404
 
         context = generate_html_tree(subtree)
-        template = context['node']['template_name']
         context['use_comments'] = True
+        template = context['node']['template_name']
 
+        context = {'sub_context': context, 'sub_template': template,
+                   'preamble': preamble}
         if not request.is_ajax() and request.GET.get('partial') != 'true':
-            # Wrap the inner context
-            context = {'sub_context': context, 'sub_template': template,
-                       'preamble': preamble}
             template = 'regulations/preamble-chrome.html'
+        else:
+            template = 'regulations/preamble-partial.html'
```

Comment:
First, isn't this how the other views work--for example, regulations? We have a top-level template (`chrome.html`) that includes a partial content template (`regulation-content.html`) that includes a node-level template (`default.html`). The markup for the section managed by the client-side `ChildView` lives in the intermediate template.

Second, best I can tell, our Backbone views want all the content managed by a `ChildView` to live inside the partial. Every time the user navigates to a new section, the `MainView` wipes out everything within `#content-body` and instantiates a new `ChildView` to manage the new markup. We can put the comment markup outside `#content-body`--but then `PreambleView` has to manage elements outside its `$el`, or we have to add another view that isn't managed like other `ChildView`s. Also, the backbone views expect `#content-wrapper` to come from the partial--which we could do with some ugliness inside a custom node template that adds that markup, but only for the root node.

By the way, I agree with your intuition here--I'm just not sure how to make it work with the frontend.